src/Input/index.scss

```diff
@@ -98,12 +98,12 @@
     pointer-events: none;
     font-family: var(--font-family-body);
     font-weight: 400;
-    font-size: 12px;
+    font-size: var(--font-size-s);
     color: var(--color-mediumGrey);
-    line-height: 1.25;
+    line-height: var(--font-lineHeight-bodyText);
```

Comment:
This is equivalent - I'm just doing some housekeeping here 🧹 

Line heights in NDS are always unitless and relative, so these values will continue to scale properly with font size adjustments.